core/src/main/resources/swagger/rawls.yaml

```diff
@@ -4616,11 +4616,11 @@ definitions:
         type: string
         description: The root entity type that the method will be running on
       prerequisites:
-        $ref: ''
+        type: object
```

Comment:
in swagger-ui 2.2 (but not in 2.1 apparently), `$ref: ''` caused the UI to think this was an external http reference, and for every instance of `$ref: ''`, it made an ajax request to '' (the root directory). This resulted in 24 useless extra ajax requests.